vector/src/main/java/im/vector/riotx/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -1108,7 +1108,14 @@ class RoomDetailFragment @Inject constructor(
                 showSnackWithMessage(getString(R.string.copied_to_clipboard), Snackbar.LENGTH_SHORT)
             }
             is EventSharedAction.Delete                     -> {
-                roomDetailViewModel.handle(RoomDetailAction.RedactAction(action.eventId, context?.getString(R.string.event_redacted_by_user_reason)))
+                val layout = requireActivity().layoutInflater.inflate(R.layout.dialog_delete_event, null)
+                AlertDialog.Builder(requireActivity())
+                        .setView(layout)
```

Comment:
Please add a title "Delete message"

Reply:
The title and content of the dialog are set as the Riot Web.